frontend/app/src/components/settings/data-security/backups/BackupManager.vue

```diff
@@ -1,202 +1,174 @@
-          message: e.message
-        }).toString()
-      });
-    } finally {
-      set(loading, false);
-    }
+const userDb = computed(() => {
+  const info = get(backupInfo);
+  return {
+    size: info ? size(info.userdb.info.size) : '0',
+    version: info ? info.userdb.info.version.toString() : '0'
   };
-  onMounted(loadInfo);
+});
+
+const globalDb = computed(() => {
+  const info = get(backupInfo);
   return {
-    backups,
-    backupInfo,
-    userDb,
-    globalDb,
-    directory,
-    loading,
-    loadInfo
+    schema: info ? info.globaldb.globaldbSchemaVersion.toString() : '0',
+    assets: info ? info.globaldb.globaldbAssetsVersion.toString() : '0'
   };
+});
+
+const { info } = useBackupApi();
+
+const loadInfo = async () => {
+  try {
+    set(loading, true);
+    set(backupInfo, await info());
+  } catch (e: any) {
+    logger.error(e);
+    notify({
+      display: true,
+      title: t('database_backups.load_error.title'),
+      message: t('database_backups.load_error.message', {
+        message: e.message
+      })
+    });
+  } finally {
+    set(loading, false);
+  }
 };
+const isSameEntry = (firstDb: UserDbBackup, secondDb: UserDbBackup) =>
+  firstDb.version === secondDb.version &&
+  firstDb.time === secondDb.time &&
+  firstDb.size === secondDb.size;
 
-const setupBackupActions = (
-  directory: Ref<string>,
-  backupInfo: Ref<DatabaseInfo | null | undefined>,
-  refresh: () => Promise<void>,
-  selected: Ref<UserDbBackup[]>
-) => {
-  const { t } = useI18n();
-  const { deleteBackup } = useBackupApi();
-  const massRemove = async () => {
-    const filepaths = get(selected).map(db => getFilepath(db, directory));
-    try {
-      await deleteBackup(filepaths);
-      if (get(backupInfo)) {
-        const info: DatabaseInfo = { ...get(backupInfo)! };
-        get(selected).forEach((db: UserDbBackup) => {
-          const index = info.userdb.backups.findIndex(backup =>
-            isSameEntry(backup, db)
-          );
-          info.userdb.backups.splice(index, 1);
-        });
-        set(backupInfo, info);
-      }
-      set(selected, []);
-    } catch (e: any) {
-      logger.error(e);
-      notify({
-        display: true,
-        title: t('database_backups.delete_error.title').toString(),
-        message: t('database_backups.delete_error.mass_message', {
-          message: e.message
-        }).toString()
-      });
-    }
-  };
+onMounted(loadInfo);
 
-  const remove = async (db: UserDbBackup) => {
-    const filepath = getFilepath(db, directory);
-    try {
-      await deleteBackup([filepath]);
-      if (get(backupInfo)) {
-        const info: DatabaseInfo = { ...get(backupInfo)! };
+const { deleteBackup } = useBackupApi();
+const massRemove = async () => {
+  const filepaths = get(selected).map(db => getFilepath(db, directory));
+  try {
+    await deleteBackup(filepaths);
+    if (get(backupInfo)) {
+      const info: DatabaseInfo = { ...get(backupInfo)! };
```

Comment:
because of how problematic this `!` can be, I usually stay away from it...
in this context, it should be fine... I'd do this anyways:

```typescript
const backup = get(backupInfo);
if (backup) {
      const info: DatabaseInfo = { ...backup };
```

Reply:
I pushed some changes to remove both instances.